bt_action_server:
  ros__parameters:
    action_name: "/bt_executor/execute_tree"
    tick_frequency: 20
    groot2_port: 1667
    ros_plugins_timeout: 1000

    plugins:
      - "robot_actions/lib"
    behavior_trees:
      - "bt_executor/trees"

    # bt_executor custom parameters
    plugin_directories:
      - "/home/luke/generalist_bt_gen/install/robot_actions/lib"
    status_topic: "/mission_coordinator/status_text"
    active_node_topic: "/mission_coordinator/active_subtree"
    auto_restart_on_failure: false
    feedback_rate_hz: 5.0
